js/projects.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var projects = [];
+
+function Project(name, startDate, endDate, description, link){
+  this.name = name,
+  this.startDate = startDate,
+  this.endDate = endDate,
+  this.description = description,
+  this.link = link,
+  projects.push(this)
+}
+var salmonCookies = new Project('Salmon Cookies', 'August 2017', 'August 2017', 'Prototype business website to track cookie sales at different store locations; Create customer-facing website advertising the business.', 'https://sedillon93.github.io/cookie-stand/');
```

Comment:
this should be in a separate file
- app.js should hold your constructor(s) 
- projects.js should hold your project data inside of an array of project objects.